accessibility-swarms.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: page
+title: Accessibility Swarms
+---
+Accessibility Swarms are a practice that we carry out in BBC News.
+
+Swarms take place on **all new front-end code**, this means any changes to HTML, CSS or JavaScript code.
+
+We do this because **accessibility is a team responsibility**.
+
+## When should you carry out a swarm?
+- Before a ticket moves from Development to Test
+- When HTML, CSS & JS is code complete
+- Some developer testing with some assistive technology should already have taken place
+
+## How many people do you need?
+### If it’s a new component or feature
+Carry out a **team swarm**, with 4 to 6 people.
+
+### If it’s a small change
+Carry out a **developer swarm**.
+
+- You don’t need 4 to 6 people
+- The developers pairing on the ticket can work through the **[checklist](accessibility-news-and-testers)** alone
```

Comment:
This assumes everyone is pairing. It'd be good to update this line and the one above to say: `After code reviewing, two developers should work through this **[checklist](accessibility-news-and-testers)**`

Reply:
Add this